src/components/settings/SettingsForm.js

```diff
@@ -17,31 +17,49 @@ function SettingsForm ({
     display: validationError ? 'block' : 'none'
   }
   return (
-    <form onSubmit={loadSpace} styleName='settings-form'>
-      <p><a href='https://contentful.com'>Contentful</a> is a content management platform for web applications, mobile apps and connected devices.</p>
-      <p>It allows you to create, edit & manage content in the cloud and publish it anywhere via a powerful API.</p>
-      <p>The Contentful Discovery web app gives you a quick and easy way to preview your content on a web environment, and explore the contents of your Spaces</p>
-      <p>You can get your Space ID and Access Token from the API section of the <a href='https://app.contentful.com'>Contentful Web App</a></p>
-      <div styleName='form-container'>
-        <p styleName='error' style={errorDisplay}>{validationError}</p>
-        <div>
-          <label for='space' styleName='label-title'>Space ID</label>
-          <input id='space' type='text' value={space} onChange={handleChange}/>
+    <div styleName="settings-container">
+      <h3 styleName="settings-title">WEB DISCOVERY APP</h3>
+      <form onSubmit={loadSpace} styleName='settings-form'>
+        <div styleName='form-container'>
+          <p styleName='error' style={errorDisplay}>{validationError}</p>
+          <div>
+            <h1>Access any Space with Space ID and API Key</h1>
+            <a href='#'>Where do I find Space ID and Production API Key</a>
+          </div>
+          <div>
+            <label for='space' styleName='label-title'>Space ID</label>
+            <input styleName='spaceInput' id='space' type='text' value={space} onChange={handleChange}/>
+          </div>
+          <div>
+            <label for='productionAccessToken' styleName='label-title'>Production API Key</label>
+            <input id='productionAccessToken' type='text' value='' onChange={handleChange}/>
+          </div>
+          <div>
+            <label for='previewAccessToken' styleName='label-title'>Preview Access Token</label>
+            <input id='previewAccessToken' type='text' value='' onChange={handleChange}/>
+          </div>
+          <button type='submit'>Load Space</button>
         </div>
-        <div>
-          <label for='selectedAccessToken' styleName='label-title'>Access Token</label>
-          <select id='api' value={selectedApi} onChange={handleChange}>
-            <option value='delivery'>Delivery API</option>
-            <option value='preview'>Preview API</option>
-          </select>
-          <input id='selectedAccessToken' type='text' value={selectedAccessToken} onChange={handleChange}/>
-        </div>
-        <button type='submit'>Load Space</button>
+      </form>
+      <div styleName="form-caption">
+        <section>
+          <h4>What is Contentful Discovery Web App?</h4>
```

Comment:
Also, perhaps you should talk with one of our designers (@hollensteiner ?) on whether we really want to capitalize some of these terms.

We just changed this is lots of places of our web app where we do not capitalize in most places anymore I believe.